### ISSUE TITLE
A11y_SQL Azure Data Studio_Add an Azure account_Add an Azure account_Reflow:On zooming screen content at 400% then  deploy a data tier radio button name is not completely visible.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/azuredatastudio#16306]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.10;
Pre-Requisite:
Set display resolution 

Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Win10

**Repro Steps:**

1. Launch the application-Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over to the 'Azure' header and select the '+' icon.
4. Hover over to the Azure header in the blade that opens up and click on the "+" icon.
5. Tab navigate to deploy a data tier radio button and observe its not visible completely on 400% zoom.

**Actual:**
On zooming screen content at 400% then  deploy a data tier radio button name is not completely visible.

**Expected:**
On zooming screen content at 400% then  deploy a data tier radio button name should be completely visible.

**User Impact:**
Low vision user will miss some information if the radio button name is not visible on zooming 400%.
![16306](https://user-images.githubusercontent.com/65954864/126135765-aea8cd1b-2cc9-488f-8d80-5899a85de09a.png)

**Comment by alanrenmsft** [comment]
closing the issue, Azure Data Studio requested exception for WCAG 2.1: https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by Ashutoshsahoo164** [comment]
GitHubTags:#Closed;
